multi-cluster-services/multi-cluster-services-basic/README.md

```diff
@@ -0,0 +1,195 @@
+```
+
+### Try it out
+
+1. Download this repo and navigate to this folder
+
+    ```sh
+    $ git clone https://github.com/GoogleCloudPlatform/gke-networking-recipes.git
+    Cloning into 'gke-networking-recipes'...
+
+    $ cd gke-networking-recipes/multi-cluster-services/multi-cluster-services-basic
+    ```
+
+2. Deploy the two clusters `gke-1` and `gke-2` as specified in [cluster setup](../cluster-setup.md)
+
+3. Now follow the steps for cluster registration with Hub and enablement of Multi-cluster Services.
+
+    There are two manifests in this folder:
+
+    - app.yaml is the manifest for the `whereami` Deployment and Service.
+    - export.yaml is the manifest for the `ServiceExport`, which will be deployed to indicate exporting service.
+
+4. Now log into `gke-1` and deploy the app.yaml manifest. You can configure these contexts as shown [here](../cluster-setup.md).
+
+    ```bash
+    $ kubectl --context=gke-1 apply -f app.yaml
+    namespace/multi-cluster-demo unchanged
+    deployment.apps/whereami created
+    service/whereami created
+
+    # Shows that pod is running and happy
+    $ kubectl --context=gke-1 get deploy -n multi-cluster-demo
+    NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+    whereami          1/1     1            1           44m
+    ```
+
+
+5. Now create ServiceExport in export.yaml to export service to other cluster.
+
+    ```bash
+    $ kubectl --context=gke-1 apply -f export.yaml
+    serviceexport.net.gke.io/whereami created
+    ```
+
+6. It can take up to 5 minutes to propagate endpoints when initially exporting Service from a cluster. Create the same Namespace in `gke-2` to indicate you want to import service, and inspect ServiceImport and Endpoints.
+
+    ```bash
+    $ kubectl --context=gke-2 create ns multi-cluster-demo
+    namespace/multi-cluster-demo created
+    
+    # Shows that service is imported and ClusterSetIP is assigned.
+    $ kubectl --context=gke-2 get serviceimport -nmulti-cluster-demo
+    NAME       TYPE           IP              AGE
+    whereami   ClusterSetIP   [10.124.4.24]   4m50s
+    
+    # Shows that endpoints are propagated.
+    $ kubectl --context=gke-2 get endpoints -nmulti-cluster-demo
+    NAME                 ENDPOINTS        AGE
+    gke-mcs-7pqvt62non   10.16.4.7:8080   6m1s
+    ```
+
+7. Now use `whereami.multi-cluster-demo.svc.clusterset.local` or `ClusterSetIP` to access endpoint from `gke-2`.
+
+    ```bash
+    $kubectl --context=gke-2 run -ti --rm --restart=Never --image=radial/busyboxplus:curl shell-$RANDOM -- curl whereami.multi-cluster-demo.svc.clusterset.local | jq -r '.zone, .cluster_name, .pod_name'
+    us-west1-a
+    gke-1
+    whereami-559545767b-xrd4h
+    ```
+
+Now to demonstrate how MCS can be used for cluster upgrade, let's simulate migration scenario. Assuming `gke-2` is the new cluster, you can deploy and export the same service from `gke-2` together from `gke-1`.
+
+1. Deploy and export the same service in `gke-2`.
```

Comment:
This step feels like it needs another diagram. It's not 100% clear that foo is deployed locally in gke-2 and also in gke-1

Reply:
Inserted a new diagram based on previous diagram.